Cargo.toml

```diff
@@ -34,7 +35,7 @@ name = "cardinality_estimator"
 harness = false
 
 [features]
-default = []
+default = ["mem_dbg"]
```

Comment:
Is it really necessary to enable `mem_dbg` feature by default?

Reply:
Only if by the end of this PR we were to replace size_of with mem_dbg - the two methods are by definition very overlapping in scope. Possibly, reading the other comments, you intended size_of to only include the size of the stored object, while mem_dbg has the goal of computing the size of the object in memory.

Reply:
If `mem_dbg` aims to show real heap memory usage similarly to `size_of` then it could be a good replacement / alternative to current `size_of` implementation.